src/GitHub.App/Services/PullRequestEditorService.cs

```diff
@@ -229,33 +247,61 @@ public class PullRequestEditorService : IPullRequestEditorService
             Guard.ArgumentNotEmptyString(relativePath, nameof(relativePath));
             Guard.ArgumentNotNull(thread, nameof(thread));
 
-            await OpenDiff(session, relativePath, thread.CommitSha);
-
-            // HACK: We need to wait here for the diff view to set itself up and move its cursor
-            // to the first changed line. There must be a better way of doing this.
-            await Task.Delay(1500);
+            var diffViewer = await OpenDiff(session, relativePath, thread.CommitSha, scrollToFirstDiff: false);
 
             var param = (object)new InlineCommentNavigationParams
             {
                 FromLine = thread.LineNumber - 1,
             };
 
-            VisualStudio.Services.Dte.Commands.Raise(
-                Guids.CommandSetString,
-                PkgCmdIDList.NextInlineCommentId,
-                ref param,
-                null);
+            // HACK: We need to wait here for the inline comment tags to initialize so we can find the next inline comment.
+            // There must be a better way of doing this.
+            await Task.Delay(1500);
+            RaiseWhenAvailable(Guids.CommandSetString, PkgCmdIDList.NextInlineCommentId, param);
```

Comment:
I thought this hack was no longer necessary with `DifferenceViewerOptions.ScrollToFirstDiffId`?

Reply:
So did I! 😢  It seems there's another issue where the `NextInlineCommentId` depends on the comment margin tags having been updated in order to navigate to the next comment. Before the tags have been refreshed, this command is unavailable and it would crash when attempting to execute the command.

We should attempt to fix this in another PR.